### PR TITLE
Remove profile

### PR DESCRIPTION
**Pull request by StachuDev**
contains two branches: userSettings removeProfile

### CODE REVIEW

File: Client/src/pages/profileSettings/ProfileSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+    });
+  }
+
+  useEffect(() => {
+    loadUserData();
+  }, []);
+
+  const resetForm = () => {
+    reset({
+      email: undefined,
```
Comment (MaksymAndriienko) [inline Client/src/pages/profileSettings/ProfileSettings.tsx]: null lepeij

File: Client/src/pages/profileSettings/ProfileSettings.tsx [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+      password: undefined,
+    });
+    setInputIsOpen(-1);
+  };
+
+  const onSubmit = async (formData: SettingsSchemaType) => {
+    const cleanedData: SettingsFormType = cleanSettingsData(formData);
+    const keys = Object.keys(cleanedData) as (keyof SettingsFormType)[];
+    const key = keys[0];
+    console.log("key", key);
```
Comment (MaksymAndriienko) [inline Client/src/pages/profileSettings/ProfileSettings.tsx]: Usuwamy wszystkie console log

File: Client/src/pages/profileSettings/ProfileSettings.tsx (hunk 1 above)
Comment (StachuDev) [reply Client/src/pages/profileSettings/ProfileSettings.tsx]: [set defalut zod validation value as null](https://github.com/MaciejMakowskiBille/NetflixClone/pull/20/commits/d67813f1ba0c15cdadece75dc151d40e9af94a9c)

File: Client/src/pages/profileSettings/ProfileSettings.tsx (hunk 2 above)
Comment (StachuDev) [reply Client/src/pages/profileSettings/ProfileSettings.tsx]: [removing console.logs](https://github.com/MaciejMakowskiBille/NetflixClone/pull/20/commits/8fbacf43a8e202077269ffb47156dbb32e068b9f)

**Review by kowalowy1234 — APPROVED** [review]

**Pull request merged**